single-line-diagram/single-line-diagram-cgmes/single-line-diagram-cgmes-layout/src/test/java/com/powsybl/sld/cgmes/layout/AbstractCgmesVoltageLevelLayoutTest.java

```diff
@@ -27,8 +27,8 @@ abstract class AbstractCgmesVoltageLevelLayoutTest {
     protected void test(VoltageLevel vl) {
         VoltageLevelGraph graph = new NetworkGraphBuilder(vl.getNetwork()).buildVoltageLevelGraph(vl.getId());
         LayoutParameters layoutParameters = new LayoutParameters();
-        layoutParameters.setScaleFactor(2);
-        layoutParameters.setDiagramName(DIAGRAM_NAME);
+        layoutParameters.setCgmesScaleFactor(2);
```

Comment:
I'm not sure here too that we should call it CGMES. As it is true that today the extension (so no automatic) layout mode onyly comes from CGMES it is the plan to move all theses extension to core and not anymore related to CGMES.

Reply:
It is a workaround I found in order not to touch this part of the code in this PR. I created an [issue](https://github.com/powsybl/powsybl-diagram/issues/510) to track that.